README.md

```diff
@@ -43,6 +43,8 @@ Our custom binaries can be found at:
 - etcdctl: https://downloads.giantswarm.io/etcd/v2.1.0-gs-1/etcdctl
 - fleet: https://downloads.giantswarm.io/fleet/v0.11.3-gs-2/fleetd
 - fleetctl: https://downloads.giantswarm.io/fleet/v0.11.3-gs-2/fleetctl
+- rkt: https://downloads.giantswarm.io/rkt/v1.1/rkt
+- kubectl: https://downloads.giantswarm.io/k8s/v1.1.8/kubectl
```

Comment:
In my opinion, we could omit it, and just specify that these binaries are downloaded from an URL with the form: `https://downloads.giantswarm.io/{COMPONENT}/{VERSION}/{BINARY_NAME}`